src/components/EventPane.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div>
+    <h1>I am event {{ $route.params.id }}</h1>
+    <pre>{{ $data }}</pre>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'event',
+  data() {
+    return {
+      event: null,
```

Comment:
I'm not super clear on why this returns event: null. Seems like a Vue thing, but is there an easy explanation?

Reply:
Yeah, this tells Vue what data the component has so that it can generate the appropriate getters/setters with observers. Later when I set `_this.event = ...`, Vue will pick up that it changed and rerender the relevant parts of the template.

It additionally serves the purpose of setting defaults for the data, but there isn't a good default in this case.